xamarin/Xamarin.Forms
# Revert "[GTK] Fix visibility of hidden pages in the stack (#3904)"

This reverts commit b5b9404bda761e7e5941da7ff26f1d42dd62d1b2.

### Description of Change ###

Regression introduced in https://github.com/xamarin/Xamarin.Forms/pull/3904 

### Issues Resolved ### 

- fixes https://github.com/xamarin/Xamarin.Forms/issues/6663

### API Changes ###

 None

### Platforms Affected ### 

- GTK

### Behavioral/Visual Changes ###

Fixes navigation issues.

### Testing Procedure ###

Try running https://gitlab.com/knocte/geewallet/commits/bugreport/gtk-navgoback2 with last 3.3 (good) or first 3.4 (bad) to see the issue.

### PR Checklist ###


/cc @knocte
